ddl/db_test.go

```diff
@@ -7236,43 +7266,6 @@ func (s *testSerialDBSuite) TestIssue22819(c *C) {
 	c.Assert(err, ErrorMatches, ".*8028.*Information schema is changed during the execution of the statement.*")
 }
 
-func (s *testSerialSuite) TestTruncateAllPartitions(c *C) {
```

Comment:
Why is ```TestTruncateAllPartitions``` mrigated to ```ddl/serial_test.go``` ? Keep the test case in the origin file.

Reply:
OK, but I'm not so sure what's the difference whether you keep testcases in the origin file...

done

Reply:
it is convenient to track historical changes with git.

Reply:
If you change the position of the code as you want, it's difficult for me to see the process of update this code directly.